src/init.cpp

```diff
@@ -20,15 +20,16 @@ void drive_PID() {
 	chassisTurn.slewOverrideEnable = true;
 	while (true) {
 		if(chassisState == NORMAL){
-			chassisStraight.setInput(left_encoder.get_value());//placeholder until encoder gets in
+			chassisStraight.setInput(left_encoder.get_value());
 			chassisStraight.moveTo();
 
-			chassisPWM(chassisStraight.getOutput(), chassisStraight.getOutput());
-		} else if(chassisState == PIVOT){
+			chassisPWM(chassisStraight.getOutput());
```

Comment:
You should revert this. It will make it easier for us to add straightening code in a little bit, which will require moving both sides at different speeds.